docs/application/web/guides/multimedia/metadata.md

```diff
@@ -0,0 +1,43 @@
+# Metadata
+
+You can extract metadata information from multimedia files.
+
+The main features of the Metadata API include:
+
+- Extraction of a metadata from a file
+  You can [extract metadata](#extract-metadata) from multimedia files using `get()` method.
+
+
+## Prerequisites
+
+To access files using the Metadata API (in [mobile](../../api/latest/device_api/mobile/tizen/metadata.html), [wearable](../../api/latest/device_api/wearable/tizen/metadata.html)), the application has to request [proper storage permissions](../security/privacy-related-permissions.md).
```

Comment:
When I check TWDAPI-265, it seems below privileges are requested.

http://tizen.org/privilege/mediastorage 
http://tizen.org/privilege/externalstorage 

If it is correct, would you please follow the style that used in other guide pages?

To access files using the Metadata API (in [mobile](../../api/latest/device_api/mobile/tizen/metadata.html),  [wearable](../../api/latest/device_api/wearable/tizen/metadata.html), and [TV]([wearable](../../api/latest/device_api/tv/tizen/metadata.html) ) applications), the application has to request permission by adding the following privileges to the `config.xml` file:
```
<tizen:privilege name="http://tizen.org/privilege/mediastorage "/>
<tizen:privilege name="http://tizen.org/privilege/externalstorage "/>
```


Or, is there any reason that you didn't follow the above form?

Reply:
Mentioned privileges are optional for applications. I added a section which describes this, besides of PPM-related part of comment.